plugin.video.cumination/resources/lib/sites/stripchat.py

```
@@ -118,66 +118,138 @@ def clean_database(showdialog=True):
+            'https://stripchat.com/api/external/v4/widget/?limit=1&modelsList={0}',
+            'https://stripchat.com/api/front/models?limit=1&modelsList={0}&offset=0'
+        ]
+        for endpoint in endpoints:
+            try:
+                response = utils._getHtml(endpoint.format(model_name), site.url, headers=headers)
+                payload = json.loads(response)
+                models = payload.get('models') if isinstance(payload, dict) else None
+                if models:
+                    return models[0]
+            except Exception:
+                continue
```

Ruff (0.14.3)

`try`-`except`-`continue` detected, consider logging the exception

(S112)